api/python/provisioner/config.py

```diff
@@ -423,13 +421,12 @@ class GroupChecks(Enum):
 
 # validations parameters
 NETWORK_DRIVER = "mlnx-ofed"
-HCA_PROVIDER = "mellanox"
-HBA_PROVIDER = "lsi"
+HCA_PROVIDER = ["mellanox"]
```

Comment:
Made the provider names as a list and looped it through the implementation, to handle in case more providers are added in future.